src/System.Management.Automation/engine/interpreter/CallInstruction.Generated.cs

```diff
@@ -121,7 +121,7 @@ internal partial class CallInstruction {
                 case TypeCode.Int32: return FastCreate<Int32>(target, pi);
                 case TypeCode.Int64: return FastCreate<Int64>(target, pi);
                 case TypeCode.Boolean: return FastCreate<Boolean>(target, pi);
-                case TypeCode.Char: return FastCreate<Char>(target, pi);
+                case TypeCode.Char: return FastCreate<char>(target, pi);
```

Comment:
I think this one shouldn't be changed. We are dealing with primitive types in general here and the official type name is more readable than alias, e.g. `UInt64` vs. `ulong`. Better use official type name for all types here.

Reply:
```suggestion
                case TypeCode.Char: return FastCreate<Char>(target, pi);
```

Reply:
```suggestion
                case TypeCode.Char: return FastCreate<Char>(target, pi);
```